### ISSUE TITLE
About rain100H Dataset

### ISSUE THREAD

**Issue by yanlongbinluck** [tsingqguo/efficientderain#20]
Hi，I have a question about dataset. For rain100H dataset，Why is there a large number of images simultaneously existing in the training set and the test set  ? Is this training on the test set?

**Comment by MKFxxL** [comment]
Thanks for your question.
The rain100H dataset we used is the same as the offcial rain100H_oldversion dataset. You can also download it from here: https://github.com/nnUyi/DerainZoo/blob/master/DerainDatasets.md
This rain100H dataset is also widely used by other people.

**Comment by tsingqguo** [comment]
We follow the official setups of Rain100H for a fair comparison. We will also check the dataset we used. However, according to the results on other datasets, we believe it has limited effects on our performance.  We will let you know any updates. Thanks and regards.
